requirements.txt

```diff
@@ -28,3 +28,5 @@ requests >= 2.25.1, < 3
 unittest-xml-reporting >= 3.0.4, < 4
 markdown ~= 3.3.4
 aplus-auth ~= 0.1.0
+celery >= 5.1.2, < 6
```

Comment:
Is there a reason to name `celery 5.1.2` here when also `celery 5.2.1` has been released?